util/include/crypto/eddsa/EdDSA.hpp

```diff
@@ -90,7 +90,7 @@ static ByteArrayKeyClass getByteArrayKeyClass(const std::string& key, concord::u
   if (KeyFormat::PemFormat == format) {
     std::memcpy(resultBytes.data(), extractHexKeyFromPem<ByteArrayKeyClass>(key, keyLength).data(), keyLength);
   } else if (KeyFormat::HexaDecimalStrippedFormat == format) {
-    std::memcpy(resultBytes.data(), concordUtils::unhex(key).data(), keyLength);
+    std::memcpy(resultBytes.data(), boost::algorithm::unhex(key).data(), keyLength);
```

Comment:
use fromHexString

Reply:
Changes in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2